server/src/main/java/org/opensearch/action/search/SearchRequestContext.java

```diff
@@ -89,7 +89,7 @@ void setShardStats(int total, int successful, int skipped, int failed) {
         this.shardStats.put(ShardStatsFieldNames.SEARCH_REQUEST_SLOWLOG_SHARD_FAILED, failed);
     }
 
-    String formattedShardStats() {
+    public String formattedShardStats() {
```

Comment:
This should not be needed, right?

Reply:
I will revert this once comment above is fixed - https://github.com/opensearch-project/OpenSearch/pull/12103#discussion_r1478906333

Reply:
Done